ledger-core/v2/cmd/pulsard/main.go

```diff
@@ -49,6 +52,10 @@ func parseInputParams() inputParams {
 	if err != nil {
 		fmt.Println("Wrong input params:", err.Error())
 	}
+	if oneShot, exists := os.LookupEnv(PulsarOneShotEnv); exists {
```

Comment:
please make it a part of configuration/or just get it from viper (viper not only parses file, it searches flags and ENV if proper ENV prefix is defined)

Reply:
done